src/components/login/ProviderLink.tsx

```diff
@@ -9,11 +9,27 @@ interface Props {
   provider: LoginProvider,
 }
 
+function loginMethod(provider: LoginProvider) {
```

Comment:
@danielsilverstone-ct this implementation now basically ignores `provider.text`. Best would probably be to move this to the backend and just have something like `provider.name` that we can use.

Reply:
OK, let's discuss this week what would be the ideal thing for the backend to return.

Reply:
just go for `provider.name` as in `Google`, `GitHub`, `GitLab` etc.
I would tread `provider.method` like an id and expect it to never change, so we can use it to decide which assets (logos) to show